PULL-REQUEST.md

```diff
@@ -0,0 +1,25 @@
+# Pull Request Template
+
+**Person who creates PR SHOULD NOT merge own pull request** 
+
+---
+## Description
+1 - 2 sentences describing purpose of pull request
+
+
+## Link to Trello Card
+Provided link [here]
+
+
+---
+## Commit Formatting
+Commits should follow the format below,
+```
+git commit -m 'FIX: readme'
+git commit -m 'FEAT: initial navbar component'
+git commit -m 'MISC: re-organizing repo for docker build'
+```
+
+## Reviewers
+- Each pull request should include a minimum of 2 reviewers
+- Reviewers should comment a minimum of 2 constructive sentences
```

Comment:
Maybe put: Last reviewer should merge if everyone has approved changes.

Reply:
Just saw this...adding this to file now 👍